setup.py

```diff
@@ -46,6 +46,8 @@ def read(fname):
         'Programming Language :: Python :: 3.6',
     ],
 
+    # support sagemaker-inference==1.1.0 for mxnet 1.4 eia image and
```

Comment:
nit: for clarity can we include the version long name (1.4.1 and 1.6.0), as we didn't make any changes to 1.4.0 ei